avocado/core/nrunner.py

```diff
@@ -679,9 +679,12 @@ def __init__(self, runnable, identifier=None, status_uris=None,
         :type status_uri: list
         :param known_runners: a mapping of runnable kinds to runners.
         :type known_runners: dict
+        :param category: category of this task. Defaults to 'test'.
+        :type category: str
         """
         self.runnable = runnable
         self.identifier = identifier or str(uuid1())
+        self.category = (category or 'test').lower()
```

Comment:
If the default is 'test', than this could be part of the method signature. `category='test'`

Reply:
I was avoiding adding it as a default value because I was planning to make some kind of automated resolution of what is a `test` and everything else later and remove this parameter.

Anyway, I don't see a problem with adding `test` as the default value now.